config/azure-api.env

```diff
@@ -0,0 +1 @@
+AZURE_API_KEY = ""
```

Comment:
There need to be comments in these env files adding instructions for how to fill in these values for non-IMAGE people.